corpus/annotations.txt

```diff
@@ -8,7 +8,7 @@ Classes
 
 (compilation_unit
   (class_definition
-    (annotation (type_identifier) (arguments (string) (string)))
+    (annotation (type_identifier) (arguments (literal (string)) (literal (string))))
```

Comment:
Let's not add this wrapper `literal` node; we generally try to avoid having these single-child wrapper nodes in order to keep the tree concise and maximally efficient to traverse.

We can still keep the rule in the grammar by either by renaming it to `_literal` so that it's "hidden", or adding `$.literal` to the grammar's `inline` array. I would prefer whichever solution gives us the lower `STATE_COUNT` in the generated parser.

Reply:
Both approaches return the same zero-value difference for `STATE_COUNT`.
I'll stick with name without an underscore and using `inline` array.